legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabSession.java

```diff
@@ -25,4 +25,12 @@ public interface GitLabSession extends Session
     void clearGitLabToken();
 
     void setGitLabToken(GitLabToken token);
+
+    void setRefreshToken(GitLabToken token);
+
+    GitLabToken getRefreshToken();
+
+    void setTokenExpiry(long expiryIn);
```

Comment:
We should make clear that we talking about seconds here.
```suggestion
    void setTokenExpiry(long expiresInSecs);
```